src/main/java/seedu/fridgefriend/command/ListCommand.java

```diff
@@ -74,27 +106,54 @@ public String getListAllMessage() {
     private String getFoodDescription(int index) {
         int indexShownToUser = index + EXTRA_INDEX;
         Food food = fridge.getFood(index);
-        String foodDescription = 
-                "\n\t" + indexShownToUser +  ". "
-                + food.getFoodName() + " ["
-                + food.getCategory() + "]";
+        String foodDescription =
+                "\n\t" + indexShownToUser + ". "
+                        + food.getFoodName() + " ["
+                        + food.getCategory() + "]";
         return foodDescription;
     }
 
-    private void checkIsValidCategory() throws InvalidInputException {
-        if (!FoodCategory.contains(category)) {
-            String errorMessage = "Sorry my friend, please enter a valid food category.";
-            throw new InvalidInputException(errorMessage);
+    private boolean checkIsValidCategory() {
+        if (FoodCategory.contains(description)) {
+            return true;
+        } else {
+            return false;
         }
     }
 
+    private boolean checkIsValidStorageLocation() {
```

Comment:
Same here, just return FoodStorageLocation.contains(descrption)

Reply:
same here, I think better to separate out

Reply:
Good catch, resolved